packages/php-db-import-export/tests/functional/Bigquery/SqlBuildTest.php

```diff
@@ -813,30 +813,23 @@ public function testGetUpdateWithPkCommandConvertValuesWithTimestamp(string $usi
 
     public function nullManipulationWithTimestampFeatures(): Generator
     {
-        yield 'import:row-change:IS_NULL' => [
-            'import:row-change:IS_NULL',
+//        yield 'default' => [
+//            'default',
+//            // phpcs:ignore
+//            'UPDATE `import_export_test_schema`.`import_export_test_test` AS `dest` SET `col1` = `src`.`col1`, `col2` = `src`.`col2`, `_timestamp` = \'2020-01-01 01:01:01\' FROM `import_export_test_schema`.`stagingTable` AS `src` WHERE `dest`.`col1` = `src`.`col1` ',
+//            true,
+//        ];
```

Comment:
Tady jsem chtěl otestovat default chování. Nicméně BQ vraci chybu `UPDATE/MERGE must match at most one source row for each target row`
Nepodařilo se mi přijít na to, proč. Dohledal jsem, že to může být duplicitami v tab.
`If a row in the table to be updated joins with more than one row from the FROM clause, then the query generates the following runtime error: UPDATE/MERGE must match at most one source row for each target row`

V té zdrojové test tabulce je duplicita, tak nevím, zda to je očekávané chování.

Reply:
huh, asi by nevadilo, ze se to tady neotestuje, protoze je to ve "normalnich testech", ale ta chyba me znervoznuje. @zajca mohl bys na to mrknout pls?

Reply:
ou to je teda blbé, to by znamenalo že bysme museli dedupnůt už zdroj aby tam nebyl žádný duplicitní řádek. Ono to dává smysl protože když ti dojde 2x stejný PK v importu tak to který z nich ti ostane není deterministické to budeme muset otestovat. Dost dobrý catch.

Reply:
jo toto sa nestane. protože sa to používá při incrementalu a tam sa udělá dedup tabulka kde sů rows nedeterministicky deduplikované a ten update ide z dedup tabulky do final. Takže tento case je pořešený.

Reply:
ok, mažu